.github/workflows/benchmark.yaml

```
@@ -120,76 +78,23 @@ jobs:
-            fi
-          else
-            echo 'No benchmark baseline found yet; skipping comparison'
-          fi
-
+          COMPARE_JSON: target/bench/workdir/compare.json
+          HEAD_SHA: ${{ github.event.pull_request.head.sha }}
+        run: bash .github/scripts/bench-report.sh
       - name: Comment results on PR
-        if: github.event.pull_request.head.repo.full_name == github.repository && hashFiles('benchmark/workdir/compare.json') != ''
+        if: github.event.pull_request.head.repo.full_name == github.repository && hashFiles('target/bench/workdir/compare.json') != ''
         uses: actions/github-script@v7
```

actionlint (1.7.12)

[error] missing input "script" which is required by action "actions/github-script@v7". all required inputs are "script"

(action)